### PR TITLE
[api] Add hashes handling for params

### PR DESCRIPTION
**Pull request by kilativ-dotcom**
* [x] Read PR [documentation](https://github.com/ostis-ai/sc-machine/blob/main/CONTRIBUTING.md)
* [x] Update changelog
* [ ] Update documentation

### CODE REVIEW

File: sc-memory/sc-memory/sc_template_build.cpp [hunk 1]
```diff
@@ -109,6 +109,15 @@ class ScTemplateBuilder
         ObjectInfo obj = CollectObjectInfo(item.second, std::to_string(addr.Hash()));
         m_elements.insert({addr.Hash(), obj});
       }
+      else
+      {
+        ObjectInfo obj = CollectObjectInfo(item.second, item.first);
```
Comment (NikitaZotov) [inline sc-memory/sc-memory/sc_template_build.cpp:114]: Please, update changelog

Comment (NikitaZotov) [reply sc-memory/sc-memory/sc_template_build.cpp:114]: And test to check this logic

**Review by NikitaZotov — CHANGES_REQUESTED** [review]

File: sc-tools/sc-builder/tests/units/test_base.cpp [hunk 2]
```diff
@@ -7,6 +7,7 @@
 #include <gtest/gtest.h>
 
 #include "builder_test.hpp"
+#include "../../src/scs_loader.hpp"
```
Comment (NikitaZotov) [inline sc-tools/sc-builder/tests/units/test_base.cpp:10]: Remove it

**Review by NikitaZotov — APPROVED** [review]

**Comment by kilativ-dotcom** [comment]
merge pls 🥹

**Pull request merged**